source/change-streams/change-streams.rst

```diff
@@ -10,7 +10,7 @@ Change Streams
 :Type: Standards
 :Minimum Server Version: 3.6
 :Last Modified: July 30, 2018
-:Version: 1.5.1
+:Version: 1.6.1
```

Comment:
Last modified date should probably also be bumped.

Reply:
Done.